Tools/baselineDef.h

```diff
@@ -40,6 +40,19 @@ class BaselineVessel
         std::string METPhiLabel = "metphi";
 
         std::string muonsFlagIDLabel = "muonsFlagMedium";
+        try
+        {
+          const double &temp  = tr.getVar<double>("genHT");
```

Comment:
there is a "ntpVersion" (std::vector<string>) stored in the Spring15 sample (probably not in PHYS14, need to check). This can be used for checking versions evolution. As a temporary solution, "genHT" is fine.